internal/pkg/mount/v2/cgroups.go

```diff
@@ -14,14 +15,15 @@ import (
 	"github.com/siderolabs/talos/pkg/machinery/constants"
 )
 
-// ForceGGroupsV1 returns the cgroup version to be used (only for !container mode).
-func ForceGGroupsV1() bool {
+// ForceCGroupsV1 returns the cgroup version to be used (only for !container mode).
+func ForceCGroupsV1() bool {
 	return pointer.SafeDeref(procfs.ProcCmdline().Get(constants.KernelParamCGroups).First()) == "0"
 }
 
 // CGroupMountPoints returns the cgroup mount points.
 func CGroupMountPoints() Points {
-	if ForceGGroupsV1() {
+	if ForceCGroupsV1() {
+		log.Println("cgroupsv1 is deprecated and will not be supported in the 1.10 release other than in the container mode")
```

Comment:
this logger is not as nice, can you please move this method to startup tasks which calls this method? (it has zap logger)